### PR TITLE
save & cancel button added on edit page

### PR DESCRIPTION
**Pull request by tasawernawaz**
[ECOM-6054](https://openedx.atlassian.net/browse/ECOM-6054)

Screenshot of buttons:

screen shot 2017-01-09 at 6 13 12 pm

### CODE REVIEW

**Review by waheedahmed — APPROVED** [review]
Looks great, nice work 👍

**Comment by tasawernawaz** [comment]
@bderusha Please review.

File: course_discovery/static/sass/publisher/publisher.scss [hunk 1]
```diff
@@ -737,3 +737,14 @@ select {
     text-decoration: underline;
   }
 }
+
+.btn-cancel {
+  padding-right: 20px;
```
Comment (bderusha) [inline course_discovery/static/sass/publisher/publisher.scss]: RTL compatibility

**Review by bderusha — APPROVED** [review]
👍 once comment addressed

**Pull request merged**